ScorchApp/src/components/dungeonMaster/DungeonMasterMain.vue

```diff
@@ -1,26 +1,41 @@
 <template>
     <div class="dm-main-component">
+        <div class="alert alert-success success-notification" id='success-msg' v-bind:class="{'notify-active' : displaySuccess}">
+            <strong>{{successMsg}}</strong>
+        </div>
+        <div class="alert alert-danger failure-notification" id="failure-msg" v-bind:class="{'notify-active' : displayError}">
+            <strong>{{errorMsg}}</strong>
+        </div>
```

Comment:
can you turn this notifications thing to a component? I think we want to use something like this everywhere

Reply:
in my mind... I think a Vue Plugin makes the most sense https://vuejs.org/v2/guide/plugins.html. 

Ideally I think it be cool if it was like... this.$notify('info', msg) or this.$notify('error', msg) or  if you can.... this.$notify.info(message) and then it would do your fadeIn/Out on the message.

Reply:
if you look at the Vue router plugin, it gives you both custom components and also custom functions